From 1eebeae565cfcc40a7ad9aad5f517b0c2431a342 Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Wed, 05 Apr 2023 10:59:10 +0300
Subject: [PATCH] [SW-66396] gaudi3: script: use fake fuse offsets from specs

This patch uses fake fuse spi offsets from specs file
gaudi3/gaudi3_mem_addr_dt.h. This will be included in all
repo's so that any change will automatically get reflected
in all repo's.

Change-Id: Ib9427d934304137b5b736d240597a88b6e89c8e2
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/board/habanalabs/gaudi3_arc/post-image-spi.sh b/board/habanalabs/gaudi3_arc/post-image-spi.sh
index 0753871..51a507e 100755
--- a/board/habanalabs/gaudi3_arc/post-image-spi.sh
+++ b/board/habanalabs/gaudi3_arc/post-image-spi.sh
@@ -19,8 +19,6 @@
 PPBOOT_OFFSET=0x20000
 PPBOOT2_OFFSET=0x40000
 PPBOOT_TEXT_BASE=0x0300007FD3140000
-FAKE_FUSE_OFFSET=0xE0000
-FAKE_FUSE2_OFFSET=0xE1000
 FAKE_FUSE_TEXT_BASE=0x0
 PPBOOT_START_ADDR=0x0300007FD3140000
 FAKE_FUSE_START_ADDR=0x0
@@ -447,6 +445,7 @@
 	local KEY_NUM=0
 	local EROM_IMG=$2
 
+	hl_create_section_variables
 	# Prepare the NEXT and MODIFIABLE for backward compatibility
 	echo "11110000" > ${HL_DEV}-next-image.txt
 	dd status=none conv=sync bs=8 count=1 if=${HL_DEV}-next-image.txt of=$img_file
